src/core/plugins/json-schema-2020-12/components/keywords/Title/Title.jsx

```diff
@@ -9,15 +9,11 @@ import { useFn } from "../../../hooks"
 
 const Title = ({ title = "", schema }) => {
   const fn = useFn()
-  const renderedTitle = title || fn.getTitle(schema)
+  const renderedTitle = fn.getTitle(schema) || title
```

Comment:
We cannot really do this. Observe how the `getTitle` is defined:

```js
export const getTitle = (schema) => {
  const fn = useFn()

  if (schema?.title) return fn.upperFirst(schema.title)
  if (schema?.$anchor) return fn.upperFirst(schema.$anchor)
  if (schema?.$id) return schema.$id

  return ""
}
```

It was designed to get some sensible title when title is not provided from the outside. So it will construct it from the `$anchor` or `$id` along with `title` keyword. You don't really want that. 

The title provided from outside should always have a priority and `getTitle` should be used only as last resort.


This change will effectively make all the schemas under `properties` (if they have `title`, `$id` or `$anchor`) use computed name from `getTitle` function instead of the real properly name.